sdk/src/main/scala/com/horizen/params/NetworkParams.scala

```diff
@@ -33,6 +34,8 @@ trait NetworkParams {
   val sidechainGenesisBlockId: ModifierId
   val sidechainGenesisBlockParentId: ModifierId = bytesToId(new Array[Byte](32))
   val signersPublicKeys: Seq[SchnorrProposition]
+  val mastersPublicKeys: Seq[SchnorrProposition]
+  val typeOfCircuitNumber: Int
```

Comment:
`val certCircuitType: CircuitType`
`cert` prefix is important, because we also have CSW circuit

Reply:
As we agreed previously, we use int instead of CircuitType as enum. Python tests are also written for int, according to the previous suggestion

Reply:
Done